### DISCUSSION TITLE
Skala Mapy

### DISCUSSION THREAD

**Discussion by jarhead99** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#273] [General]
Hej!
mam pytanie , udało mi się wgrać integrację - świetna robota. Niestety problem mam ponieważ nie mogę obrócić mapy o 180, ani nie potrafię dodać np stref. Takie rzeczy edytuje się w  configuration.yaml czy bezpośrednio w karcie ? 
<img width="751" alt="Zrzut ekranu 2021-12-26 o 23 44 44" src="https://user-images.githubusercontent.com/90874863/147421664-31dc0974-f04e-4f1d-89b7-71cb4cd9ccf5.png">

**Comment by PiotrMachowski** [comment]
Obraz mapy się modyfikuje w konfiguracji kamery (configuration.yaml), strefy trzeba dodać w konfiguracji karty

**Reply by jarhead99** [reply]
dziękuję!  sorry za pewnie banalne pytania , ale dopiero poznaje HA:)))

**Comment by jarhead99** [comment]
a jest jakiś tutorial dla bardzo początkujących?:) co gdzie, jak wpisać? bo absolutnie akurat mi nie idzie;)

**Reply by PiotrMachowski** [reply]
W obydwóch Readme są przykłady konfiguracji, powinny wystarczyć
